conduit/v1/fromproto/schema.go

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fromproto
+
+import (
+	"fmt"
+
+	"github.com/conduitio/conduit-commons/schema"
+	conduitv1 "github.com/conduitio/conduit-connector-protocol/proto/conduit/v1"
+)
+
+func SchemaInstance(req *conduitv1.CreateSchemaRequest) (schema.Instance, error) {
```

Comment:
I propose to introduce types for `CreateSchemaRequest` (and other requests and responses) in `github.com/conduitio/conduit-connector-protocol/conduit/v1`, to follow the same pattern we use in `cplugin` (for example [here](https://github.com/ConduitIO/conduit-connector-protocol/blob/4230db1b457029e74c11f252e4019a5eccd25543/cplugin/source.go#L58-L61)). Even if the type is empty or just contains a single field, it makes the code explicit. If we set up a strict structure now I believe it will be easier to work with this down the road. Currently for example it's not exactly clear that we only got 3 fields in the request, since `schema.Instance` contains more fields than that.

That said, in the create response we can reference a `schema.Instance` in the struct, since the proto message reflects exactly the same type (and that shouldn't change).

Btw, we'll also need proto conversions for responses, we might as well add them now already. We'll need them in the SDK.

Reply:
Done!